pkg/kds/v2/server/components.go

```diff
@@ -50,7 +50,7 @@ func New(
 		util_xds_v3.AdaptDeltaCallbacks(util_xds.LoggingCallbacks{Log: log}),
 		util_xds_v3.AdaptDeltaCallbacks(statsCallbacks),
 		// util_xds_v3.AdaptDeltaCallbacks(NewNackBackoff(nackBackoff)),
-		newKdsRetryForcer(log, cache, hasher), // this change might requires backoff to be enabled
```

Comment:
So does it require backoff? How do we know?

Reply:
It would be good to introduce backoff. Because delta xDS doesn't have retry and sometimes K8s store update might fail, we are responding with NACK. This is not a failure and we want to retry it. But sometimes NACK might be a real issue and it's difficult to distinguish the real reason, so we retry all NACKs. In case of config error, we might DDOS control-plane with NACKs.

Reply:
don't we have exponential backoff with retries?

Reply:
if we don't turn nack on then the next successful update will "catch up" right?

Reply:
Backoff is not enabled yet because of unstable tests that require investigation. When first ACK arrives NACK are not sent so everything should be ok.